lib/libc/Kconfig

```diff
@@ -678,3 +678,10 @@ config LIB_USRWORKSTACKSIZE
 
 endif # LIB_USRWORK
 endif # BUILD_PROTECTED || BUILD_KERNEL
+
+
+config AUDIO
```

Comment:
Audio framework has 3 parts, I2S driver, Audio driver in os/drivers/audio/Kconfig. Framework mostly a library is currently placed in libc. When Taejun decides, it might be moved to Framework directory. But for now, it is in libc.